changelogs/unreleased/6202-repair-loop.yml

```diff
@@ -0,0 +1,6 @@
+description: Prevent repairs from restarting indefinitely when a short deploy interval is set
+issue-nr: 6202
+change-type: minor
```

Comment:
Shouldn't this be a patch?

Reply:
I was in doubt myself, it slightly changes the behavior. 

I you think patch is more appropriate, I will change it.

Reply:
I would have chosen patch here, but I also don't have any objections against using minor instead.

Reply:
> I would have chosen patch here, but I also don't have any objections against using minor instead.

I side with this comment.